test/integration/integration_test.go

```diff
@@ -169,10 +170,10 @@ func FeatureContext(s *godog.Suite) {
 		getRoutingUrlAndVerifyHTTPResponse)
 
 	// steps for verifying config file content
-	s.Step(`JSON config file "([^"]*)" (contains|does not contain) key "(.*)" with value "(.*)"$`,
-		configContains)
-	s.Step(`JSON config file "([^"]*)" (contains|does not contain) key "(.*)"(.*)$`,
-		configContains)
+	s.Step(`^JSON config file "([^"]*)" (contains|does not contain) key "(.*)" with value "(.*)"$`,
+		jsonConfigContains)
+	s.Step(`^JSON config file "([^"]*)" (has|does not have) key "(.*)" with value assigned$`,
```

Comment:
Any specific reason we want to add `^` i.e.  the sentence should start with  `JSON config file`?